consensus/consensus.md

```diff
@@ -134,11 +134,14 @@ nonce:
 #### Key Blocks
 
 ```
-PROTOCOL_VERSION: 2
-GENESIS_VERSION: 2
+PROTOCOL_VERSION: 21
+GENESIS_VERSION: 21
 ```
 
-- the timestamp of a block MUST be smaller than `now() + 30m` TODO: re-calc for NG
+- the timestamp of a key block MUST be smaller than `now() + 10m`
```

Comment:
May I suggest relaxing this MUST to SHOULD? `now()` is a relative concept so the same block in 10 minutes will be valid. It seems only thing possible is not unsolicitedly forwarding block.

Reply:
I see - no need for changing MUST to SHOULD. Given a local `now()`, a block is valid or not depending on this. How to actually handle blocks that will eventually become valid is an implementation detail.